action.yml

```
@@ -3,8 +3,10 @@ description: 'A GitHub Action to deploy an artifact to GitHub Pages'
 runs:
   using: 'node16'
   main: 'dist/index.js'
-  pre: 'pre/index.js'
 inputs:
+  emit_telemetry:
+    description: 'Should this action emit build telemetry instead of deploy the build artifact'
+    required: false
```

```suggestion
    required: false
    default: true
```
should we add default value here? or we pass it from dynamic workflow? :)
()

---

ah did not read the pr description... my bad 🙈